lib/workers/bin/run_single_worker.rb

```diff
@@ -27,6 +27,10 @@
     options[:guid] = val
   end
 
+  opts.on("-e=ems_id", "--ems-id=ems_id,ems_id", Array, "Provide a list of ems ids (without spaces) to a provider worker. This requires, at least one argument.") do |val|
```

Comment:
Mr. Fancypants using Array.  Someone read the optparse documentation.  😎 
Nice @juliancheal

Reply:
I admit to that catching me off guard when I first read that as well.

Reply:
@jrafanie Hehehe, thanks!